source/Symbol/SwiftASTContext.cpp

```diff
@@ -3432,11 +3433,13 @@ swift::ASTContext *SwiftASTContext::GetASTContext() {
 
   // 2. Create and install the module interface loader.
   //
-  // TODO: It may be nice to reverse the order between MIL and SML in
-  //       LLDB, since binary swift modules likely contain private
-  //       types that the module interfaces are missing. On the
-  //       other hand if we need to go looking for a module on disk,
-  //       something is already screwed up in the debug info.
+  // This order is different from the Swift compiler, where the
+  // serialized module loader comes before the module interface
```

Comment:
Hm, in swiftc the module interface loader _does_ come before the serialized loader...